src/julia-parser.scm

```diff
@@ -1406,11 +1406,11 @@
        ((function macro)
         (let* ((paren (eqv? (require-token s) #\())
                (sig   (parse-def s (not (eq? word 'macro)))))
-          (if (and (eq? word 'function) (not paren) (symbol-or-interpolate? sig))
+          (if (and (not paren) (symbol-or-interpolate? sig))
               (begin (if (not (eq? (require-token s) 'end))
-                         (error (string "expected \"end\" in definition of function \"" sig "\"")))
+                       (error (string "expected \"end\" in definition of " word " \"" sig "\"")))
```

Comment:
Sorry to hound you about this but the indentation is still wrong.

Reply:
No problem can you explain how the indentation works? This is what vim suggests and also what I thought was right. It is one level deeper then the above `(if` so indent by two spaces?

Reply:
Ok it needs to align with the `(not`, now I get it.